website/content/docs/common-workflows/manage-users-groups.mdx

```diff
@@ -283,7 +283,7 @@ Now you can test logging in.
 
 
 ```bash
-$ boundary authenticate password -login-name myuser -auth-method-id ampw_PbE6nNT72a
+$ boundary authenticate password -auth-method-id ampw_PbE6nNT72a
```

Comment:
while there's no problem with the changes here, it should be noted that all the `/docs/common-workflows/` have been hidden from the navigation, and are no longer maintained. The navigation link instead redirects to the tutorials now.

These workflows should probably be fully deprecated going forward, since we recently discovered hidden TOC items still show up in dev dot search results